### DISCUSSION TITLE
verify publish source for Oracle consumer contract error KeyError: 'local'

### DISCUSSION THREAD

**Discussion by ptn77** [smartcontractkit/full-blockchain-solidity-course-py#1384] [Q&A]
I'm deploying an smart contract that consumes the QDT Data Provider and it deploys successfully, but when it tries to verify and publish the source, I'm getting an error. 
The Oracle data provider information is as follows:
Ethereum Kovan Testnet
Payment Amount: 0.1 LINK
LINK Token Address: [0xa36085F69e2889c224210F603D836748e7dC0088](https://kovan.etherscan.io/address/0xa36085F69e2889c224210F603D836748e7dC0088)
Oracle Address: 0xfF07C97631Ff3bAb5e5e5660Cdf47AdEd8D4d4Fd
JobID: 31c4d01ad1f549eca40c560d47efa870

The following is the code to deploy:

api_consumer = QDTChainlink.deploy(
        oracle,
        Web3.toHex(text=jobId),
        fee,
        link_token,
        {"from": account},
        publish_source=config["networks"][network.show_active()].get("verify"),
    ) 

The following is the error:

Transaction sent: 0x2badc51b3be918fa6db42f94ebb8fde3c0f7e1a1f14eb7aecbd3d766408de2b2
  Gas price: 2.500000007 gwei   Gas limit: 906714   Nonce: 67
  QDTChainlink.constructor confirmed   Block: 30925569   Gas used: 824286 (90.91%)
  QDTChainlink deployed at: 0xF35cEFDCC916E46d22cabc9EE1926E170419C654

  File "brownie/_cli/run.py", line 49, in main
    return_value, frame = run(
  File "brownie/project/scripts.py", line 103, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File "./scripts/01_deploy_api_consumer.py", line 39, in main
    deploy_api_consumer()
  File "./scripts/01_deploy_api_consumer.py", line 17, in deploy_api_consumer
    api_consumer = QDTChainlink.deploy(
  File "brownie/network/contract.py", line 600, in __call__
    return tx["from"].deploy(
  File "brownie/network/account.py", line 555, in deploy
    contract.publish_source(deployed_contract, silent=silent)
  File "brownie/network/contract.py", line 410, in publish_source
    contract_info = self.get_verification_info()
  File "brownie/network/contract.py", line 292, in get_verification_info
    if symbol_alias["local"] is not None:
KeyError: 'local'

Has anyone seen this error before? Any help is appreciated.

**Comment by cromewar** [comment]
Hello @ptn77 
Here is some tips for you:

1. Look for any indentation error on your brownie config file.
2. be sure the name of your etherscan API key is: `ETHERSCAN_TOKEN`.
3. On brownie config the verify statement should be with upper case: True.

I hope any of this info might help.
